http/servlet-undertow/src/test/java/io/quarkus/ts/http/undertow/UndertowMissingServletClassIT.java

```diff
@@ -0,0 +1,51 @@
+package io.quarkus.ts.http.undertow;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.bootstrap.Service;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.scenarios.annotations.DisabledOnQuarkusVersion;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+@DisabledOnQuarkusVersion(version = "3\\.(8|9|10|11|12|13|14)\\..*|3\\.15\\.(0|1)(\\..*)?", reason = "Disabled on Quarkus versions before 3.15.2 due to build fail because missing servlet class configuration undertow issue https://github.com/quarkusio/quarkus/issues/43825")
```

Comment:
I think we don't need this, if the build fails and the app is not hanging, the test will fail. Can you check if we even need it? 

It fail with `Caused by: org.opentest4j.AssertionFailedError: Resource failed to start` inside of `assertDoesNotThrow` so I think there should be some better message when this `DisabledOnQuarkusVersion` is removed


In previous case the app was hanging for 3 or 5 min and left the open port, so from my POV this are the types of cases which should be disabled on older versions